File: templates/micronaut/files/src/main/java/sh/platform/template/HelloController.java
package sh.platform.template;

import io.micronaut.http.MediaType;
import io.micronaut.http.annotation.Controller;
import io.micronaut.http.annotation.Get;
import io.reactivex.Single;

@Controller("/")
public class HelloController {

    @Get(uri = "/", produces = MediaType.TEXT_PLAIN)
    public Single<String> hello() {
        return Single.just("Hello World! A simple Micronaut template for Platform.sh");
    }
}

File: templates/spring-kotlin/files/src/main/kotlin/sh/platform/template/spring/kotlin/HelloController.kt
package sh.platform.template.spring.kotlin

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class HelloController() {

    @GetMapping("/")
    fun helloKotlin(): String {
        return "hello world from Platform.sh"
    }
}